ablNeutralEdge/setup.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env
+"""
+Run this to generate the input files
+You will need aprepro in the path when you execute
+
+'python setup.py' will generate the ablNeutralEdge cases
+'python setup.py /absolute/path/to/build/dir' will
+generate the actuatorLine cases
+"""
+import sys
+import os
+import subprocess
+
+mesh_size = ["02", "05", "10", "20", "40"]
+time_step = [0.15,  0.3,  0.6,  1.2,  2.4]
+cases = []
+
+input_file = "input.yaml"
+output_file = "{base}_{ms}m_sgs2_CFL_485.yaml"
+
+if len(sys.argv) > 1:
+    use_actuator = True
+    base_name = "nrel5MWactuatorLine"
+    nrel5mw_base_dir = os.path.join(sys.argv[1],"reg_tests/test_files/nrel5MWactuatorLine")
+    time_step_scaling = 1e-1
+
+    with open(os.path.join(nrel5mw_base_dir, '5MW_Baseline','NRELOffshrBsline5MW_Onshore_ServoDyn.dat'), 'r') as source:
+        fname = 'ServoDyn.dat'
+        with open(fname, 'w') as target:
+            data = source.read()
+            data = data.replace(r"5   PCMode", r"0   PCMode")
+            data = data.replace(r"2   GenModel", r"1   GenModel")
+            data = data.replace(r"5   VSContrl", r"1   VSContrl")
+            data = data.replace(r"9999.9   VS_RtGnSp", r"1161.96   VS_RtGnSp")
+            data = data.replace(r"9999.9   VS_RtTq", r"43093.55   VS_RtTq")
+            data = data.replace(r"9999.9   VS_Rgn2K", r"0.025576   VS_Rgn2K")
+            data = data.replace(r"9999.9   VS_SlP", r"10.0   VS_SlP")
+            target.write(data)
+else:
+    use_actuator = ""
```

Comment:
It would make more sense to set `use_actuator = False` since in the if branch it is set to `use_actuator = True`, this would also avoid the later comparison with the empty string.